### PR TITLE
Overhaul README

### PR DESCRIPTION
**Pull request by pat-s**
Long overdue.

I applied some subjective changes, feedback welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,55 +40,55 @@
...
 
-Here are some places where people mention Woodpecker:
+Woodpecker is used as the main CI/CD engine at [Codeberg](https://codeberg.org), an alternative Git hosting platform with a focus on privacy and free software development.
+While there are people which contribute to both projects, there is no official relationship between both.
 
-- [GitHub](https://github.com/topics/WoodpeckerCI)
-- [Codeberg](https://codeberg.org/explore/repos?q=woodpeckerci&topic=1)
-- [Twitter](https://twitter.com/search?q=%23WoodpeckerCI&src=typed_query)
-- [Fediverse](https://mastodon.social/tags/WoodpeckerCI)
+While Woodpecker hosts the core repos at GitHub (for historical reasons and outreach), the offical plugins are developed in the [woodpecker-plugins organization at Codeberg](https://codeberg.org/woodpecker-plugins).
```
Comment (zc-devs) [inline README.md:83]: What's about [Kaniko plugin](https://woodpecker-ci.org/plugins/Kaniko)?

Comment (zc-devs) [reply README.md:83]: That is good, but the issue is open and there is no kaniko repo on Codeberg.

File: README.md [hunk 2]
```diff
@@ -7,9 +7,6 @@
 </p>
 <br/>
 <p align="center">
-  <a href="https://ci.woodpecker-ci.org/repos/3780" title="Pipeline Status">
```
[the remark this answers is not in the document]
Comment (pat-s) [reply README.md:10]: It is mainly "no status" and looks weird/not helpful. This state comes from the "pending" runs that are not activated.

Do we have any benefit from keeping it?

File: README.md [hunk 3]
```diff
@@ -43,55 +40,55 @@
...
+We have a self-hosted [Weblate](https://weblate.org/en/) instance at [translate.woodpecker-ci.org](https://translate.woodpecker-ci.org).
 
-Woodpecker is used by [itself](https://ci.woodpecker-ci.org/woodpecker/woodpecker-ci/), multiple well-known companies, organizations like [Codeberg](https://codeberg.org), hobbyists and many others.
+An overview of the current translation state is available at <https://translate.woodpecker-ci.org/projects/woodpecker-ci/#languages>.
 
-Leave a [comment](https://github.com/woodpecker-ci/woodpecker/discussions/2149) if you're using it as well.
+To help with translation, the language-specific files in <https://github.com/woodpecker-ci/woodpecker/tree/main/web/src/assets/locales> can be edited.
 
-Also consider using the topic `WoodpeckerCI` in your repository, so others can learn from your config and use the hashtag `#WoodpeckerCI` when talking about the project on social media!
+## Woodpecker and Codeberg
```
[the remark this answers is not in the document]
Comment (pat-s) [reply README.md:78]: The title should only indicate the content of the following paragraph. How would you phrase it?

**Comment by pat-s** [comment]
> Can you remove the red/orange box around the screenshot?

This is the outline of my browser. But I can redo the screenshot with another browser.

File: README.md (hunk 1 above)
Comment (pat-s) [reply README.md:83]: The goal was not too go into too much detail WRT to plugins. Maybe adding https://woodpecker-ci.org/plugins in addition to the org link is more clear?

Comment (zc-devs) [reply README.md:83]: > in addition

Instead, I would say. Until all repos are migrated.
Or two links: github, codeberg.

Comment (pat-s) [reply README.md:83]: We then forget to update this part again. There are only a few plugins on GH anyway, and linking the overall plugin registry page lets users find all sources.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (pat-s) [reply README.md]: Did you see the changes I did earlier?

**Pull request merged**